lib/ocpp/v201/charge_point.cpp

```diff
@@ -2783,6 +2783,7 @@ void ChargePoint::handle_data_transfer_req(Call<DataTransferRequest> call) {
     DataTransferResponse response;
     const auto vendor_id = msg.vendorId.get();
     const auto message_id = msg.messageId.value_or(CiString<50>()).get();
+    // first try the callbacks that are explicitly registered for a vendorId or messageId
```

Comment:
I can't find any place where the specific `data_transfer_callbacks` are set. Am I missing something or is the code block below redundant?

Reply:
You're correct, this was totally redundant code that I've removed